channel/consensus_channel/leader_channel.go

```diff
@@ -70,26 +65,31 @@ func (c *LeaderChannel) Propose(add Add, sk []byte) (SignedProposal, error) {
 	return signed, nil
 }
 
-// UpdateConsensus iterates through the leader's proposal queue until it finds a proposal
-// matching their proposal. If their signature was the follower:
-// - the consensus state is updated
+// UpdateConsensus iterates through the proposal queue until it finds the countersigned proposal.
+// If this proposal was signed by the Follower:
+// - the consensus state is updated with the supplied proposal
 // - the proposal queue is trimmed
 //
-// If their proposal is stale (ie. theirP.TurnNum <= c.current.TurnNum) then
-// their proposal is ignored
-func (c *LeaderChannel) UpdateConsensus(theirP SignedProposal) error {
-	vars := Vars{
+// If the countersupplied is stale (ie. proposal.TurnNum <= c.current.TurnNum) then
+// their proposal is ignored.
+//
+// An error is returned if:
+// - the countersupplied proposal is not found
+// - or if it is found but not correctly by the Follower
```

Comment:
Another Nit: Bullet points like this don't really work :-/ They end up getting in-lined in the documentation https://stackoverflow.com/questions/52744468/godoc-documentation-not-outputting-lists